Dispatchfile

```diff
@@ -211,6 +235,8 @@ action(tasks=["test-helm2", "test-helm3"], on=pull_request(chatops=["test"]))
 action(tasks=["test-helm2"], on=pull_request(chatops=["test-helm2"]))
 action(tasks=["test-helm3"], on=pull_request(chatops=["test-helm3"]))
 action(tasks=[do_bump_kommander_catalog_api], on=pull_request(chatops=["bump-catalog-api"]))
+action(tasks=[do_bump_grafana], on=pull_request(chatops=["bump-grafana"]))
 
 # Cron Actions
 action(name="bump-catalog-api", tasks=[do_bump_kommander_catalog_api], on=cron(schedule="0 8 * * 5"))
```

Comment:
Can you move this bump to https://github.com/mesosphere/kommander-catalog-api to trigger on tag so we don't have to poll anymore.

Reply:
https://github.com/mesosphere/kommander-catalog-api/pull/43/